core/pkg/opaprocessor/processorhandler.go

```diff
@@ -227,6 +227,11 @@ func (opap *OPAProcessor) processControl(ctx context.Context, control *reporthan
 	allResources := make(map[string]workloadinterface.IMetadata, heuristicAllocResources)
 
 	for i := range control.Rules {
+		if opap.ExcludedRules != nil {
```

Comment:
we need to decide if we use this map to excluded irrelevant rules or change the Policies object in the first place (that will require a deep copy of the Policies object)